packages/cli/src/lifecycles/bundle.js

```diff
@@ -300,11 +271,10 @@ async function bundleApiRoutes(compilation) {
 
   if (apiConfigs.length > 0 && apiConfigs[0].input.length !== 0) {
     console.info("bundling API routes...");
-    for (const configIndex in apiConfigs) {
-      const rollupConfig = apiConfigs[configIndex];
+    await asyncForEach(apiConfigs, async (rollupConfig) => {
```

Comment:
I double-checked, and `apiConfigs` is an array anyways, so converting from `for...in` to `for...of` works fine. No special handling was needed for accessing by index.